frontend/src/components/authentication/RegistrationForm.jsx

```diff
@@ -1007,14 +1008,80 @@ class RegistrationForm extends Component {
               <h3>
                 {LOCALIZE.authentication.createAccount.privacyNoticeDialog.privacyNoticeStatement}
               </h3>
-              <p>{LOCALIZE.authentication.createAccount.privacyNoticeDialog.privacyParagraph1}</p>
-              <p>{LOCALIZE.authentication.createAccount.privacyNoticeDialog.privacyParagraph2}</p>
+              <p>
+                {LOCALIZE.formatString(
```

Comment:
@fnormand01 @MichaelCherry This is an example of how to localize a string with a link or other formatting in it.